content/2021/kola/mobilnesystemyinteligentne.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Mobilne systemy inteligentne
+website: http://msi.pb.edu.pl/
+video: https://bit.ly/mobilne_inteligentne_systemy
+baseLocation: Wydział Informatyki
```

Comment:
tutaj się zastanów czy baseLocation nie powinno być kluczem jakimś absolutnym ew nazwa pliku/folderu wydzialu tak żeby łatwo to było potem zparsować